<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1021</width>
      <height>601</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QSlider" name="slider">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>0</y>
        <width>91</width>
        <height>471</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>480</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>171</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Graph</string>
      </property>
      <widget class="QPushButton" name="btn_graph1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>flow rate</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_graph2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>pressure</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_graph3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>position</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_graph4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>speed</string>
       </property>
       <property name="autoRepeatDelay">
        <number>50</number>
       </property>
       <property name="autoRepeatInterval">
        <number>50</number>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>0</y>
        <width>20</width>
        <height>511</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>171</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Modes</string>
      </property>
      <widget class="QPushButton" name="btn_mode1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Mode1</string>
       </property>
       <property name="autoRepeatDelay">
        <number>50</number>
       </property>
       <property name="autoRepeatInterval">
        <number>50</number>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_mode2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Mode2</string>
       </property>
       <property name="autoRepeatDelay">
        <number>50</number>
       </property>
       <property name="autoRepeatInterval">
        <number>50</number>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_mode3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Mode3</string>
       </property>
       <property name="autoRepeatDelay">
        <number>50</number>
       </property>
       <property name="autoRepeatInterval">
        <number>50</number>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_mode4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Mode4</string>
       </property>
       <property name="autoRepeatDelay">
        <number>50</number>
       </property>
       <property name="autoRepeatInterval">
        <number>50</number>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>0</y>
        <width>711</width>
        <height>511</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
     <widget class="QPushButton" name="btn_start">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_stop">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Calibration&amp;Setting</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
